docs/content/en/docs/fleet-manager/distributedstorage-plugin.md

```diff
@@ -0,0 +1,379 @@
+
+1. Kubernetes v1.22 or higher is supported
+1. To configure the Ceph storage cluster, at least one of these local storage types is required:
+
+   - Raw devices (no partitions or formatted filesystems)
+   - Raw partitions (no formatted filesystem)
+   - LVM Logical Volumes (no formatted filesystem)
+   - Persistent Volumes available from a storage class in `block` mode
+
+The easiest way to do this is to mount an Raw disk on the nodes.
+
+### 3. Secrets and Setup for Attached Clusters
+
+In Kurator, clusters not created by Kurator are called AttachedClusters. Kurator provides the ability to incorporate these AttachedClusters into the kurator fleet management and implement unified distributed storage on these attachedclusters with fleet. Therefore we need a fleet that already manages several AttachedClusters. Specific operations can be referred to [Manage AttachedCluster](https://kurator.dev/docs/fleet-manager/manage-attachedcluster/).
+
+## Create a Fleet with the DistributedStorage Plugin Enabled
+
+Run following command to create rook operator and rook ceph cluster in the Fleet:
+
+```console
+kubectl apply -f -<<EOF
+apiVersion: fleet.kurator.dev/v1alpha1
+kind: Fleet
+metadata:
+  name: quickstart
+  namespace: default
+spec:
+  clusters:
+    - name: kurator-member1
+      kind: AttachedCluster
+    - name: kurator-member2
+      kind: AttachedCluster
+  plugin:
+    distributedStorage:
+      storage:
+        dataDirHostPath: /var/lib/rook
+        monitor:
+          count: 3
+          labels:
+            role: MonitorNodeLabel
+        manager:
+          count: 2
+          labels:
+            role: ManagerNodeLabel
+EOF
+```
+
+### Fleet DistributedStorage Plugin Configuration Explained
+
+Let's delve into the `spec` section of the above Fleet:
+
+- `clusters`: Contains the two `AttachedCluster` objects created earlier, indicating that the distributedstorage plugin will be installed on these two clusters.
+- `plugin`: The `distributedStorage` indicates the configuration of a distributedstorage plugin. `dataDirHostPath` defines the directory in which the rook-ceph cluster will save the cluster configuration. `monitor` and `manager` provide configuration for the mon and mgr components of ceph. For more configuration options, please refer to the [Fleet API](https://kurator.dev/docs/references/fleet-api/).
+- In addition to the configuration fields mentioned above, kurator also provides `storage.devices` field that allows you to specify the use of disks mounted in a specific directory such as `sda` in the cluster. And `storage.nodes` field that allows you to specify the use of storage resources on specific nodes. For more configuration options, please refer to the [Fleet API](https://kurator.dev/docs/references/fleet-api/).
```

Comment:
I am not sure what you mean by `sda` in the cluster? Is it set in the example?

Reply:
sda is a device mounted on `/dev/sda`. if a specific mount path is specified, rook will check the specified path on all nodes in the cluster for the presence of storage resources that comply with requirements.